### ISSUE TITLE
Package breaking upon rojo sync (installed through wally)

### ISSUE THREAD

**Issue by SpiralAPI** [imezx/Warp#9]
![image](https://github.com/imezx/Warp/assets/45252038/87f67c07-4ec1-423f-92a3-a83a84030232)
![image](https://github.com/imezx/Warp/assets/45252038/0317ad1d-63ec-432e-ad9c-031f1dae27a9)
![image](https://github.com/imezx/Warp/assets/45252038/88139bf0-4362-41fb-bd35-40ed79fbdf74)

not quite sure the cause, however below is an debug log dump for rojo sync
[rojo_debug.txt](https://github.com/imezx/Warp/files/14860119/rojo_debug.txt)

**Comment by SpiralAPI** [comment]
hey, played around a bit and sourced the error down to your rojo config

```json
{
  "name": "Warp",
  "tree": {
    "$path": "src"
  }
}
```

for general safe practice in case one of your users doesn't have `ignoreUnknownInstances` set to true, i'd also update the wally config to the following

```toml
[package]
name = "imezx/warp"
version = "1.0.9"
registry = "https://github.com/UpliftGames/wally-index"
realm = "shared"
license = "MIT"
exclude = [
    "node_modules",
    "docs",
    ".github",
    "*.rbxl",
    "*.rbxmx",
    "*.rbxml"
]
description = "A very-fast & powerful networking library for Roblox."

[dependencies]
```

**Comment by imezx** [comment]
u can do a pull request for that

**Comment by imezx** [comment]
hey what rojo version u use? u need atleast ^7.4.0 for that.

**Comment by imezx** [comment]
warp use 7.4.0 and im using 7.4.1

**Comment by imezx** [comment]
look at `aftman.toml`

**Comment by SpiralAPI** [comment]
> warp use 7.4.0 and im using 7.4.1

I am using 7.4.1 as well, and wally package version 1.0.9 (imezx/warp@1.0.9)

**Comment by SpiralAPI** [comment]
> u can do a pull request for that

Done!

**Comment by imezx** [comment]
marged

**Comment by iceeburr** [comment]
#10

**Comment by iceeburr** [comment]
Until there is a new release that fixes this bug (>1.0.9), use `warp-reformat = "spiralapi/warp-reformat@2.0.1"` on Wally. (credits to spiralapi of course)
